packages/components/.storybook/manager-head.html

```diff
@@ -1 +1,123 @@
-<link href="./images/sds-favicon.png" rel="icon" type="image/png" />
+<link
+  rel="shortcut icon"
+  href="https://solid-design-system.fe.union-investment.de/docs_add-meta-information/storybook/favicon/favicon.ico"
+/>
+<link
+  rel="apple-touch-icon"
+  sizes="57x57"
+  href="https://solid-design-system.fe.union-investment.de/docs_add-meta-information/storybook/favicon/apple-icon-57x57.png"
+/>
+<link
+  rel="apple-touch-icon"
+  sizes="60x60"
+  href="https://solid-design-system.fe.union-investment.de/docs_add-meta-information/storybook/favicon/apple-icon-60x60.png"
+/>
+<link
+  rel="apple-touch-icon"
+  sizes="72x72"
+  href="https://solid-design-system.fe.union-investment.de/docs_add-meta-information/storybook/favicon/apple-icon-72x72.png"
+/>
+<link
+  rel="apple-touch-icon"
+  sizes="76x76"
+  href="https://solid-design-system.fe.union-investment.de/docs_add-meta-information/storybook/favicon/apple-icon-76x76.png"
```

Comment:
We should avoid fixed URLs in anyway, especially to external CDN resources etc.

We're already using dynamic URLs here – doesn't that work here and if not, how do they resolve?

https://github.com/solid-design-system/solid/blob/e1c4e606a5bf4c65658445877ba4a5d19a63e624/packages/components/.storybook/preview-head.html#L16

https://github.com/solid-design-system/solid/blob/e1c4e606a5bf4c65658445877ba4a5d19a63e624/packages/components/.storybook/preview-head.html#L16

Reply:
I tried with different ways to avoid fixed URLs but for some reason it only works for the favicon files. The meta tags will resolve the relative URL into `https://solid-design-system.fe.union-investment.de/sds-signet_meta.png`.

As I did not want to mix the URL type in this file I just used to latest branch CDN... If you have any idea how to solve this, I am happy to use relative URLs.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base#open_graph

This seems to be an Open Graph-Problem. So before merging this, I would suggest to point this to our x.x.x branch but leave the favicon stuff etc as it was as only Open Graph needs absolute paths.

Reply:
yes. The plan was to change it back to x.x.x as soon as I merge it. The feature branch is just for testing reasons.

I thought of only having it for the og:tags but then decided to rather have one way of using links to prevent confusion. An alternative could be a comment but I did not want to write a comment into the storybook head to explain why we mixed the URL types. 

Any idea?

Reply:
Well, for Favicons the version before was correct, for OG apparently the other one. So I would rather write a comment instead of altering to a less correct version.